op-chain-ops/genesis/config.go

```diff
@@ -761,6 +751,9 @@ type LegacyDeployConfig struct {
 	// DeploymentWaitConfirmations is the number of confirmations to wait during
 	// deployment. This is DEPRECATED and should be removed in a future PR.
 	DeploymentWaitConfirmations int `json:"deploymentWaitConfirmations"`
+
+	UnusedChannelTimeout        uint64 `json:"channelTimeout,omitempty"`
```

Comment:
We may need to honor the legacy value as an override if anybody depends on it as non standard